lib/main.coffee

```diff
@@ -189,6 +191,13 @@ module.exports = Hydrogen =
             @kernelManager.restartRunningKernelFor grammar, (kernel) =>
                 @onKernelChanged kernel
 
+        else if command is 'shutdown-kernel'
+            @clearResultBubbles()
+            # Note that destroy alone does not shut down a WSKernel
+            kernel.shutdown()
+            @kernelManager.destroyRunningKernelFor grammar
```

Comment:
Please no. The current situation exists to avoid accidental data loss, which is a major concern for me.

If I'm just temporarily attaching to a long-running kernel hosted elsewhere, I don't want the detach operation to forcibly kill the kernel! (`destroyRunningKernelFor` is called anytime you switch between kernels.)

Please let me know if there's any additional comments I can add to the code to make sure that people refactoring the code in the future are aware of this.